src/App.scss

```diff
@@ -0,0 +1,146 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter&display=swap");
+* {
+	font-family: "Open Sans";
+	font-style: normal;
+	font-weight: 400;
+	color: #080338;
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+
+tr {
+	padding-bottom: 20px;
+}
+td {
+	&:first-child {
+		padding-right: 50px;
+	}
+}
+
+.app {
+	max-width: 1340px;
+	height: 100%;
+	margin: 0 auto;
+	padding: 0;
+	&__title {
```

Comment:
all styles for each component should be in their own component folder. for example, src/components/MainTemperatureBlock should have two files 1) index.hs 2) index.scss